### ISSUE TITLE
Chromecast and subtitles issues

### ISSUE THREAD

**Issue by Tovarichtch** [bbougot/Popcorn#208]
Hello again, it's me again !

I'm on Popcorn time 5.0.1. The new UI is nice and sexy, the buffering has been fixed, the subtitles play too... except for Chromecast. It's not showing, once again. Before 5.x, it was showing !

I found something else too, but a lot less important : changing subtitles language doesn't work at all. It's not showing up, only pausing the video. But the subtitles are working into the PT player.

**Comment by Tovarichtch** [comment]
I just tried again and ... it works. I can't explain why. A bad loading ? I have sadly no clue. If I encounter this bug again, I will tell you about it.

**Comment by Jaso444** [comment]
The subtitle selection works in the menu after clicking watch, but the subtitle selector in the player doesn't work and does nothing if you click on different languages.

**Comment by bbougot** [comment]
Yes I'm aware of the issue, I'm waiting for the next release of https://github.com/unosquare/ffmediaelement

**Comment by leossmith** [comment]
Is there an update on this?
